client/state/current-user/gravatar-status/actions.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import request from 'superagent';
+
+import {
+	GRAVATAR_UPLOAD_RECEIVE,
+	GRAVATAR_UPLOAD_REQUEST,
+	GRAVATAR_UPLOAD_REQUEST_SUCCESS,
+	GRAVATAR_UPLOAD_REQUEST_FAILURE
+} from 'state/action-types';
+
+export function uploadGravatar( file, bearerToken, email ) {
+	return dispatch => {
```

Comment:
Leaving it in for now. If we feel more strongly about consistency around this, we could add a lint rule ([#](http://eslint.org/docs/rules/arrow-parens)) in the future.